src/modules/sub-items/components/table-view/table.view.item.component.js

```diff
@@ -172,7 +172,7 @@ export default class TableViewItemComponent extends PureComponent {
         const notAvailableLabel = Translator.trans(/*@Desc("N/A")*/ 'content_type.not_available.label', {}, 'sub_items');
         const contentType = contentTypesMap[content.ContentType._href];
         const contentTypeIdentifier = contentType ? contentType.identifier : null;
-        const contentTypeName = contentTypeIdentifier ? window.eZ.adminUiConfig.contentTypeNames[contentTypeIdentifier] : notAvailableLabel;
+        const contentTypeName = contentTypeIdentifier ? eZ.helpers.contentType.getContentTypeName(contentTypeIdentifier) : notAvailableLabel;
```

Comment:
In React components we do not pass `window.eZ` as `eZ` so this should stay as `window.eZ`. It also applies to other components.